lec07-pop-models.Rmd

```diff
@@ -1,4 +1,617 @@
 ---
-title: "Qualitative population models in 2 dimensions"
+title: "Population models in two dimensions and numerically solving equations in R"
+author: "Madeleine Bonsma-Fisher"
+output: html_document
 ---
 
+> ### Learning objectives
+>
+> - Review differential equations and one-dimensional phase portraits
+> - Numerically solve differential equations in R using Euler's method and R's ODE solver
+> - Qualitatively analyze two-dimensional models by hand and in R
+
+> ### Lesson outline
+> 
+> Total lesson time: 2 hours
+> 
+> - 
+
+> ### Setup
+> 
+> - `install.packages('deSolve')`
```

Comment:
Because you are using a new package, you can do one of two things in order for it to work on the website:

- Run `devtools::use_package("deSolve")` in the console (install devtools if you don't have it)
- Open up DESCRIPTION file and add `deSolve` to the line right below `Imports:`

Reply:
I forgot to say you also need to add `phaseR` to the `Imports:`

Reply:
I'll do that for next lecture - I don't think I'll get to it tomorrow.

Reply:
To using phaseR, I mean.

Reply:
Why `devtools` instead of regular `install.packages` (which seems to have worked)?